NoSoft-SA/nspack
# Feature/color percentages

Summary of this change:
added ability to capture fruit color percentages

CHANGELOG entry:

### Added
- color_applies to commodities table and CRUD
- color_percentages table and  CRUD
- color_percentage_id to product_setups, run_cache, carton_labels and pallet_sequences

### Changed
- . On commodities edit screen - show color percentages list (inline edit color_percentage)


How to test this code via the UI:

Masterfiles | Fruit | Commodities
Steps
- edit a commodity and set color_applies to true
- add color percentages to a commodity : color percentages list
- edit a commodity and inline edit color percentage (value should be between 1 and 100)

Production | Product Setups | Product Setups
- edit a product setup to add color percentage (shown only if commodity.color_applies)
 
Production | Runs | Production Runs
- allocate product setups and execute run 
- color_percentage_id will now be part of the run cache and pushed down to carton labels and pallet sequences on carton_labeling and palletizing respectively

Production | Reworks | Single Edit Pallet
- color_percentage_id will now be part of the edit pallet sequence screen
